base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
```

Comment:
I see you are setting a different width here. I tried removing it and what I see is that you could achieve the same result setting some padding.
This would be more flexible, as it would work also in different screen sizes

Reply:
Hi Namir, thats  a leftover of my first attempt. Since opacity is 0% i havent seen the old setting. THX

width 400px  (removed)

full responsive user story is following